src/mediarithmics/api/datamart/UserActivityInterface.ts

```diff
@@ -134,6 +134,7 @@ export type EventName =
     | '$email_complaint'
     | '$set_user_profile_properties'
     | '$content_corrections'
+    | string
```

Comment:
Shouldn't there be  a `type ManagedEventName= '$xxx' | '$yyy' ` with all the known eventName  and `type EventName = ManagedEventName | string` ? this would allow to constrain some function only to the known events …